src/postgres/setConnectedDatabase.ts

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ext } from "../extensionVariables";
+import { PostgresDatabaseTreeItem } from "./tree/PostgresDatabaseTreeItem";
+
+export function setConnectedDatabase(treeItem: PostgresDatabaseTreeItem | undefined): void {
```

Comment:
I don't think you need to make a command for it since it's only called once by `connectPostgresDatabase` and all it really does is just check if `connectedPostgresDB` exists in the `extensionVariables`.  Also, it shares a function name of `postgresCodeLensProvider` which confused me 😕